liberty-archetype-webapp/src/test/resources/projects/it/goal.txt

```diff
@@ -0,0 +1 @@
+clean install
```

Comment:
You'll want this too for the other archetypes, but I just noticed that you added this to the `goal.txt` in `liberty-archetype-webapp`, not `liberty-archetype-mp`.

Reply:
Oh dear.  Good catch :( .  Let me fix that.  I'm only going to do it for the mp archetype and raise an issue for the others.